conf/documentation.conf

```diff
@@ -955,6 +955,37 @@ description=<<EOT
 Color of the success text for pfcmd
 EOT
 
+[advanced.use_omapi_to_lookup_mac]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Use omapi to query dhcpd for the mac address of an ip
+EOT
+
+[advanced.omapi_key_name]
+type=text
+description=<<EOT
+The omapi key name for signing messages
+EOT
+
+[advanced.omapi_key_base64]
+type=text
+description=<<EOT
+The omapi base64 key for signing messages
```

Comment:
Should be added to the .spec to be triggered on install and added a a default value